google/cloud/pubsub/internal/subscription_session_test.cc

```diff
@@ -21,10 +21,19 @@
 #include "google/cloud/pubsub/testing/fake_streaming_pull.h"
 #include "google/cloud/pubsub/testing/mock_subscriber_stub.h"
 #include "google/cloud/pubsub/testing/test_retry_policies.h"
+#include "google/cloud/opentelemetry_options.h"
 #include "google/cloud/testing_util/async_sequencer.h"
 #include "google/cloud/testing_util/fake_completion_queue_impl.h"
 #include "google/cloud/testing_util/scoped_log.h"
 #include "google/cloud/testing_util/status_matchers.h"
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+#include "google/cloud/internal/opentelemetry.h"
+#include "google/cloud/testing_util/opentelemetry_matchers.h"
```

Comment:
nit: for consistency, only put `<opentelemetry/*.h>` includes in the `#ifdef` blocks.

Reply:
I see what you mean and changed it, but it is only used in the ifdef part, so I don't know if the nit has value besides consistency.

Reply:
I find consistency valuable. :shrug:

Reply:
Sure, but the decision could go either way right? 

We could put only what we use in the ifdef or only otel stuff. Both are valid.